Base/Python/slicer/util.py

```diff
@@ -156,7 +156,7 @@ def showStatusMessage(message, duration = 0):
   mw.statusBar().showMessage(message, duration)
   return True
 
-def findChildren(widget=None, name="", text="", title="", className=""):
```

Comment:
We could set the qt object name based on the parameter name. It would be a very simple and reliable method for finding widgets.
@ihnorton Would you be interested in implementing that? It's probably just adding a few lines in qSlicerCLIModuleUIHelper::createTagWidget.